### PR TITLE
Bug 1146296 - Add IdentityPopup.open() as helper method

### PR DESCRIPTION
**Pull request by gbrmachado**

### CODE REVIEW

File: firefox_puppeteer/tests/test_toolbars.py [hunk 1]
```diff
@@ -223,8 +223,7 @@ def tearDown(self):
 
     @skip_under_xvfb
     def test_elements(self):
-        self.locationbar.identity_box.click()
-        Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
+        self.open()
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_toolbars.py]: the `open` method is part of the `IdentityPopup` class but not of the current test function.

File: firefox_puppeteer/ui/toolbars.py [hunk 2]
```diff
@@ -436,6 +436,12 @@ def close(self, force=False):
 
         Wait(self.marionette).until(lambda _: not self.is_open)
 
+    def open(self):
+        """Opens the identity popup
+        """
+        self.locationbar.identity_box.click()
```
Comment (whimboo) [inline firefox_puppeteer/ui/toolbars.py]: `locationbar` is not a property of the IdentityPopup class. What you may want is to add the method to the LocationBar class as `open_identity_popup()`.

**Comment by whimboo** [comment]
Is there a reason why you closed this PR? You can still continue without having to create a new one. :)

**Comment by gbrmachado** [comment]
Well, I tried to reset my branch, and then I pushed it to the server. 
After that, github closed the PR. :(
I didn't know that could happen.

**Comment by whimboo** [comment]
Interesting and i haven't seen that yet myself. But ok, now we have a new PR. :)

**Pull request closed without merging**